mods/boats/init.lua

```diff
@@ -224,10 +224,10 @@ minetest.register_craftitem("boats:boat", {
 
 	on_place = function(itemstack, placer, pointed_thing)
```

Comment:
I can't see placver used anywhere in that function. pointed thing is being passed already.
But irrelevent because only end variables can be removed.